### PR TITLE
Fixes EvaluateJS crash

### PR DESCRIPTION
**Pull request by mortoray**
fixes https://github.com/fusetools/fuselibs-public/issues/262

I'd like to add a test case to the ManualTestPlan for EvaluateJS before merging this.

This PR contains:
- [-] Changelog - hasn't made a release
- [-] Documentation - not relevant
- [x] Tests - a manual test (no WebView possible in unit tests)

### CODE REVIEW

File: Source/Fuse.Controls.WebView/EvaluateJS.uno [hunk 1]
```diff
@@ -105,20 +105,24 @@ namespace Fuse.Triggers.Actions
...
 
 		public event JSEventHandler Handler;
 
+		IWebView _target;
 		public IWebView WebView {
 			get { return _target; }
 			set { _target = value; }
 		}
-
+
```
Comment (kusma) [inline Source/Fuse.Controls.WebView/EvaluateJS.uno]: Adding these tabs seems pointless.

File: Source/Fuse.Controls.WebView/EvaluateJS.uno [hunk 2]
```diff
@@ -136,7 +140,7 @@ namespace Fuse.Triggers.Actions
 		{
 			var webView = _target ?? target.FindByType<IWebView>();
 
-			if (webView != null && _rawSource != "")
+			if (webView != null && _rawSource != null)
```
Comment (kusma) [inline Source/Fuse.Controls.WebView/EvaluateJS.uno]: `if (webView != null && string.IsNullOrEmpty(_rawSource))`?

**Comment by kusma** [comment]
Do we need this on release-1.2?

File: Source/Fuse.Controls.WebView/EvaluateJS.uno (hunk 1 above)
Comment (mortoray) [inline Source/Fuse.Controls.WebView/EvaluateJS.uno]: There's no way to see these changes while editing. I also don't make them intentionally, they just happen as I move around the code.

Comment (kusma) [inline Source/Fuse.Controls.WebView/EvaluateJS.uno]: @mortoray: `git add -p` is your friend here.

**Pull request merged**